utilities/data.js

```diff
@@ -0,0 +1,4 @@
+const getInnerText = (cells) => cells.map((cell) => cell.innerText);
```

Comment:
Do not forget about Upper Case for const

Reply:
done